modules/dreamview/frontend/src/styles/main.scss

```diff
@@ -661,6 +661,68 @@ body {
         }
     }
 
+    .battery-and-gears-text {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+
+        float: right;
+        transform: translate(-50%, -50%);
+        text-align: center;
+        font-size: 14px;
+        color: #FFFFFF;
+     }
+
+    .electricity-status {
```

Comment:
1. now the electricity-status class and gear class are almost identical, except the top. Can we try consolidating the two styles? Would be great to reduce repeated code and follow reusability guideline

2. might want to move gear into the same row as battery's.